fiat-web/src/main/java/com/netflix/spinnaker/fiat/config/DefaultResourcePermissionConfig.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2019 Google, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.fiat.config;
+
+import com.netflix.spinnaker.fiat.model.resources.Account;
+import com.netflix.spinnaker.fiat.model.resources.Application;
+import com.netflix.spinnaker.fiat.model.resources.BuildService;
+import com.netflix.spinnaker.fiat.providers.*;
+import java.util.List;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+class DefaultResourcePermissionConfig {
+
+  @Bean
+  @ConditionalOnProperty(
+      value = "auth.permissions.source.account.resource.enabled",
+      matchIfMissing = true)
+  ResourcePermissionSource<Account> accountResourcePermissionSource() {
+    return new AccessControlledResourcePermissionSource<>();
+  }
+
+  @Bean
+  @ConditionalOnProperty(
+      value = "auth.permissions.provider.account",
+      havingValue = "default",
+      matchIfMissing = true)
+  public ResourcePermissionProvider<Account> defaultAccountPermissionProvider(
+      List<ResourcePermissionSource<Account>> sources) {
+    return new AggregatingResourcePermissionProvider<>(sources);
+  }
+
+  @Bean
+  @ConditionalOnProperty(
+      value = "auth.permissions.source.application.front50.enabled",
+      matchIfMissing = true)
+  ResourcePermissionSource<Application> front50ResourcePermissionSource(
+      FiatServerConfigurationProperties fiatServerConfigurationProperties) {
+    return new Front50ApplicationResourcePermissionSource(
+        fiatServerConfigurationProperties.getExecuteFallback());
+  }
+
+  @Bean
+  @ConditionalOnProperty(
+      value = "auth.permissions.provider.application",
+      havingValue = "default",
+      matchIfMissing = true)
+  public ResourcePermissionProvider<Application> defaultApplicationPermissionProvider(
+      List<ResourcePermissionSource<Application>> sources) {
+    return new AggregatingResourcePermissionProvider<>(sources);
+  }
+
+  @Bean
+  @ConditionalOnProperty(
+      value = "auth.permissions.source.account.resource.enabled",
```

Comment:
should be `build-service` instead of `account`

Reply:
thanks!